tests/upgrades/test_foreman_maintain.py

```diff
@@ -26,37 +26,109 @@ class ScenarioForemanMaintain(TestCase):
     Test Steps:
 
     1. Before Satellite upgrade, Perform test for
-       foreman-maintain list-versions.
+       foreman-maintain upgrade list-versions.
     2. Upgrade satellite/capsule.
-    3. Perform tests for foreman-maintain list-versions, after upgrade.
+    3. Perform tests for foreman-maintain upgrade list-versions, after upgrade.
     4. Check if tests passed.
 
     """
 
+    @staticmethod
+    def satellite_upgradable_version_list():
+        """
+        This function is used to collect the details of satellite version  and upgradable
+        version list.
+
+        :return: satellite_version, upgradeable_version, major_version_change
+        """
+
+        cmd = "rpm -q satellite > /dev/null && rpm -q satellite --queryformat=%{VERSION}"
+        satellite_version = ssh.command(cmd)
+        if not satellite_version.return_code:
+            satellite_version = satellite_version.stdout
```

Comment:
Please update the condition as discussed .

Reply:
Fixed